config/internal/base/deployment.yaml.tmpl

```diff
@@ -45,8 +45,6 @@ spec:
         app.kubernetes.io/instance: modelmesh-controller
         app.kubernetes.io/managed-by: modelmesh-controller
         app.kubernetes.io/name: modelmesh-controller
-        # Required in Watson Cloud envs to allow outbound access to etcd
-        network-policy: allow-egress
```

Comment:
@njhill Want to make sure, would this affect kserve users of model-mesh?

Reply:
@anhuong I don't think so